src/commands/aoc.ts

```diff
@@ -0,0 +1,63 @@
+import fetch from 'node-fetch';
+
+import { getConfig } from '../config';
+import type { Command } from '../types';
+
+const LEADERBOARD_URL = 'https://adventofcode.com/2022/leaderboard/private/view/756840';
+const AOC_SESSION = getConfig('ADVENT_OF_CODE_SESSION');
+
+// eslint-disable-next-line functional/no-let
+let leaderboard: readonly LeaderboardEntry[] = [];
+// eslint-disable-next-line functional/no-let
+let lastUpdateTimestamp = 0;
```

Comment:
To jest niepotrzebne. Mamy już paczkę `node-cache` w projekcie. Trzeba stworzyć tu nową instancję, ustawić `stdTTL` na `60 * 1000 * 15` i `deleteOnExpire` na `true`.

Reply:
Zmieniłem metodę cachowania w #[dc20b0a](https://github.com/typeofweb/typeofweb-discord-bot/pull/102/commits/dc20b0a7736ed09cb67888e35b1d592a49e071fb)